presto-orc/src/main/java/com/facebook/presto/orc/metadata/CompressionKind.java

```diff
@@ -15,5 +15,5 @@
 
 public enum CompressionKind
 {
-    UNCOMPRESSED, ZLIB, SNAPPY
+    UNCOMPRESSED, ZLIB, SNAPPY, ENCRYPTED
```

Comment:
This conflicts with the specification of the ORC format: https://orc.apache.org/docs/file-tail.html